src/compounds/tabs/src/index.tsx

```diff
@@ -166,7 +166,10 @@ export const Tabs: React.FC<TabsProps> = ({ children, className }) => {
   }, [])
 
   return (
-    <Container className={className}>
+    <Container
+      className={className}
+      sx={{ variant: 'compounds.collectionTabs.variants.containerPadding' }}
```

Comment:
`containerPadding` should just be `container` as it could hold more styles than just padding